lib/shippo/api/operations/batch.rb

```diff
@@ -0,0 +1,38 @@
+module Shippo
+  module API
+    module Operations
+      module Batch
+        # Retrieves a Batch by its ID
+        # @param [String] id The ID of the Batch object
+        # @param [Hash] params Optional params tacked onto the URL as URI parameters
+        def get(id, params={})
```

Comment:
"object_results" is part of the response. Do you mean something like "default_servicelevel_token", "label_filetype", "metadata" instead?